src/api/resources/index.ts

```diff
@@ -300,9 +305,14 @@ export class ResourcesApi {
 
           debug('req.body')
           debug(req.body)
-          if (!validate.resource(req.params.resourceType, req.body)) {
-            res.status(400).json(Responses.invalid)
-            return
+          try {
+            validate.resource(
+              req.params.resourceType as SignalKResourceType,
+              req.method,
+              req.body
+            )
+          } catch (e) {
+            res.status(400).send(e.message)
```

Comment:
I think that `.json()` should be used instead of `.send()` to be consistent with all other Signal K responses.